lib/rules/web/admin_console/4.10/logs.xyaml

```diff
@@ -13,15 +13,15 @@ switch_to_container:
 wait_log_window_loaded:
   elements:
   - selector:
-      xpath: //div[contains(@class,'log-window')]/div[contains(@class,'log-window__header')]
+      xpath: //div[contains(@class,'log-window__header')]
     timeout: 30
   - selector:
-      xpath: //div[contains(@class,'log-window')]/div[contains(@class,'log-window__body')]
+      xpath: //div[contains(@class,'pf-c-log-viewer__main')]
```

Comment:
seems it should be `//span[contains(@class,'pf-c-log-viewer__text`)] ?

Reply:
span is for each line, body is for the whole log. I think "//div[contains(@class,'pf-c-log-viewer__main')]" is better.